site/content/guides/gateway-api.md

```diff
@@ -39,80 +39,158 @@ The following prerequisites must be met before using Gateway API with Contour:
 
 ## Deploying Contour with Gateway API
 
-### Option #1: Gateway API with Contour
+Contour supports two modes of provisioning for use with Gateway API: **static** and **dynamic**.
 
-Refer to the [contour][6] design for additional background on the Gateway API implementation.
+In **static** provisioning, the platform operator defines a `Gateway` resource, and then manually deploys a Contour instance corresponding to that `Gateway` resource. It is up to the platform operator to ensure that all configuration matches between the `Gateway` and the Contour/Envoy resources.
```

Comment:
should there be some mention of the GatewayClass controller string? since the example for Option 1 uses that method

Reply:
Yeah good call, let me work that in.